tests/test_mqttcm_webcfg.c

```diff
@@ -154,6 +154,7 @@ void test_sendRbusEventWebcfgOnSubscribe_success()
 	rbusError_t ret = sendRbusEventWebcfgOnSubscribe();
 	CU_ASSERT_EQUAL(ret, 0);
 	rbushandleclose(WEBCFG_MQTT_SUBSCRIBE_CALLBACK);
+	sleep(1);//It is taking some time to unsubscribe above event, so adding delay
```

Comment:
Add rbus is taking time to unscribe the event . same for other places

Reply:
updated comment